janusgraph-core/src/main/java/org/janusgraph/core/JanusGraphFactory.java

```diff
@@ -104,7 +116,60 @@ public static JanusGraph open(BasicConfiguration configuration) {
      * @return JanusGraph graph database
      */
     public static JanusGraph open(ReadConfiguration configuration) {
-        return new StandardJanusGraph(new GraphDatabaseConfiguration(configuration));
+        return open(configuration, null);
+    }
+
+    public static JanusGraph open(ReadConfiguration configuration, String backupName) {
+        final ModifiableConfiguration config = new ModifiableConfiguration(ROOT_NS, (WriteConfiguration) configuration, BasicConfiguration.Restriction.NONE);
+        final String graphName = config.has(GRAPH_NAME) ? config.get(GRAPH_NAME) : backupName;
+        if (null != graphName) {
+            JanusGraphManager jgm = JanusGraphManager.getInstance();
+            Preconditions.checkState(jgm != null, JANUS_GRAPH_MANAGER_EXPECTED_STATE_MSG);
+            return (JanusGraph) jgm.openGraph(graphName, (gName) -> {
+                return new StandardJanusGraph(new GraphDatabaseConfiguration(configuration));
+            });
+        } else {
+            log.info("You should supply \"graph.graphname\" in your .properties file configuration if you are opening " +
```

Comment:
This is more than just the `jgm != null` case. 

The whole point of this changeset is so that the JanusGraph GremlinServer has a better way to dynamically access graphs. Users of course could previously access graphs, but these weren't tying into the TinkerPop GraphManager; the point of this changeset is to fix that odd access pattern. Therefore this log message is to act as the first steps of deprecation to accessing graphs dynamically using this old pattern.